### PR TITLE
Update connect with us section

### PR DESCRIPTION
**Pull request by kelsonic**
## Description
This PR updates the `Connect with us` section styling for CLP.

## Testing done
Locally

## Screenshots
image

## Acceptance criteria
- [x] Update connect with us section styling for CLP

## Definition of done
- [x] Events are logged appropriately
- [x] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/site/layouts/campaign_landing_page.drupal.liquid [hunk 1]
```diff
@@ -317,12 +317,51 @@
     </div>
 
     <!-- Connect with us -->
+    {% assign socialLinksObject = fieldClpConnectWithUs.entity.fieldSocialMediaLinks.platformValues | jsonToObj %}
```
Comment (ncksllvn) [inline src/site/layouts/campaign_landing_page.drupal.liquid:320]: I wonder why this wasn’t JSON to begin with

File: src/site/layouts/campaign_landing_page.drupal.liquid [hunk 2]
```diff
@@ -317,12 +317,51 @@
...
+
     <div class="vads-u-background-color--primary-alt-lightest vads-u-padding-bottom--1">
       <div class="usa-grid usa-grid-full vads-u-padding-y--4 vads-u-padding-x--2 large-screen:vads-u-padding-x--0">
         <p class="vads-u-color--gray-medium vads-u-font-size--sm vads-u-margin-bottom--0">CONNECT WITH US</p>
         <h2 class="vads-u-margin--0">Get updates from Veterans Health Administration</h2>
 
-        <!-- TODO, check out fieldClpConnectWithUs, I don't think it has everything we need but most of it should be there -->
+        <ul class="usa-unstyled-list vads-u-padding-top--4 vads-u-display--flex vads-u-flex-wrap--wrap">
+          <li class="vads-l-col--6 medium-screen:vads-l-col--4 vads-u-margin-bottom--2">
```
Comment (ncksllvn) [inline src/site/layouts/campaign_landing_page.drupal.liquid:328]: Well done with the grid classes

**Review by ncksllvn — APPROVED** [review]
LGTM

File: src/site/layouts/campaign_landing_page.drupal.liquid (hunk 1 above)
Comment (kelsonic) [reply src/site/layouts/campaign_landing_page.drupal.liquid:320]: ya it's like that all over the place, it's a good question for the cms team

File: src/site/layouts/campaign_landing_page.drupal.liquid [hunk 3]
```diff
@@ -317,12 +317,51 @@
...
+            <a href="{{ fieldClpConnectWithUs.entity.fieldEmailUpdatesUrl }}" onclick="recordEvent({ 'event': 'nav-hub-rail', 'nav-path': 'Connect with us' });" rel="noreferrer noopener" target="_blank">
+              <i class="fas fa-envelope vads-u-padding-right--1"></i>
```
Comment (timwright12) [inline src/site/layouts/campaign_landing_page.drupal.liquid:330]: Can we add `aria-hidden="true"` to icons?

File: src/site/layouts/campaign_landing_page.drupal.liquid [hunk 4]
```diff
@@ -317,12 +317,51 @@
...
+
     <div class="vads-u-background-color--primary-alt-lightest vads-u-padding-bottom--1">
       <div class="usa-grid usa-grid-full vads-u-padding-y--4 vads-u-padding-x--2 large-screen:vads-u-padding-x--0">
         <p class="vads-u-color--gray-medium vads-u-font-size--sm vads-u-margin-bottom--0">CONNECT WITH US</p>
         <h2 class="vads-u-margin--0">Get updates from Veterans Health Administration</h2>
 
-        <!-- TODO, check out fieldClpConnectWithUs, I don't think it has everything we need but most of it should be there -->
+        <ul class="usa-unstyled-list vads-u-padding-top--4 vads-u-display--flex vads-u-flex-wrap--wrap">
+          <li class="vads-l-col--6 medium-screen:vads-l-col--4 vads-u-margin-bottom--2">
+            <a href="{{ fieldClpConnectWithUs.entity.fieldEmailUpdatesUrl }}" onclick="recordEvent({ 'event': 'nav-hub-rail', 'nav-path': 'Connect with us' });" rel="noreferrer noopener" target="_blank">
```
Comment (timwright12) [inline src/site/layouts/campaign_landing_page.drupal.liquid:329]: Links the open in a new window need to announce as such. In this instance, probably something like `aria-label="[add the link text here] (opens in a new window)"`

File: src/site/layouts/campaign_landing_page.drupal.liquid [hunk 5]
```diff
@@ -317,12 +317,51 @@
...
+              <i class="fas fa-envelope vads-u-padding-right--1"></i>
+              {{ fieldClpConnectWithUs.entity.fieldEmailUpdatesLinkText }}
+            </a>
+          </li>
+          <li class="vads-l-col--6 medium-screen:vads-l-col--4 vads-u-margin-bottom--2">
+            <a href="https://twitter.com/{{ socialLinksObject.twitter }}" onclick="recordEvent({ 'event': 'nav-hub-rail', 'nav-path': 'Connect with us' });" rel="noreferrer noopener" target="_blank">
```
Comment (timwright12) [inline src/site/layouts/campaign_landing_page.drupal.liquid:335]: same new window comment

File: src/site/layouts/campaign_landing_page.drupal.liquid [hunk 6]
```diff
@@ -317,12 +317,51 @@
...
+              <i class="fab fa-twitter vads-u-padding-right--1"></i>
```
Comment (timwright12) [inline src/site/layouts/campaign_landing_page.drupal.liquid:336]: same icon comment

File: src/site/layouts/campaign_landing_page.drupal.liquid [hunk 7]
```diff
@@ -317,12 +317,51 @@
...
+              {{ fieldClpConnectWithUs.entity.name }} Twitter
+            </a>
+          </li>
+          <li class="vads-l-col--6 medium-screen:vads-l-col--4 vads-u-margin-bottom--2">
+            <a href="https://facebook.com/{{ socialLinksObject.facebook }}" onclick="recordEvent({ 'event': 'nav-hub-rail', 'nav-path': 'Connect with us' });" rel="noreferrer noopener" target="_blank">
```
Comment (timwright12) [inline src/site/layouts/campaign_landing_page.drupal.liquid:341]: same new window comment

File: src/site/layouts/campaign_landing_page.drupal.liquid [hunk 8]
```diff
@@ -317,12 +317,51 @@
...
+              <i class="fab fa-facebook vads-u-padding-right--1"></i>
```
Comment (timwright12) [inline src/site/layouts/campaign_landing_page.drupal.liquid:342]: same icon comment

File: src/site/layouts/campaign_landing_page.drupal.liquid [hunk 9]
```diff
@@ -317,12 +317,51 @@
...
+              {{ fieldClpConnectWithUs.entity.name }} Facebook
+            </a>
+          </li>
+          <li class="vads-l-col--6 medium-screen:vads-l-col--4 vads-u-margin-bottom--2">
+            <a href="https://youtube.com/channel/{{ socialLinksObject.youtube }}" onclick="recordEvent({ 'event': 'nav-hub-rail', 'nav-path': 'Connect with us' });" rel="noreferrer noopener" target="_blank">
```
Comment (timwright12) [inline src/site/layouts/campaign_landing_page.drupal.liquid:347]: same new window comment

File: src/site/layouts/campaign_landing_page.drupal.liquid [hunk 10]
```diff
@@ -317,12 +317,51 @@
...
+              <i class="fab fa-youtube vads-u-padding-right--1"></i>
```
Comment (timwright12) [inline src/site/layouts/campaign_landing_page.drupal.liquid:348]: same icon comment

File: src/site/layouts/campaign_landing_page.drupal.liquid [hunk 11]
```diff
@@ -317,12 +317,51 @@
...
+              {{ fieldClpConnectWithUs.entity.name }} YouTube
+            </a>
+          </li>
+          <li class="vads-l-col--6 medium-screen:vads-l-col--4 vads-u-margin-bottom--2">
+            <a href="https://linkedin.com/{{ socialLinksObject.linkedin }}" onclick="recordEvent({ 'event': 'nav-hub-rail', 'nav-path': 'Connect with us' });" rel="noreferrer noopener" target="_blank">
```
Comment (timwright12) [inline src/site/layouts/campaign_landing_page.drupal.liquid:353]: same new window comment

File: src/site/layouts/campaign_landing_page.drupal.liquid [hunk 12]
```diff
@@ -317,12 +317,51 @@
...
+              <i class="fab fa-linkedin vads-u-padding-right--1"></i>
```
Comment (timwright12) [inline src/site/layouts/campaign_landing_page.drupal.liquid:354]: same icon comment

File: src/site/layouts/campaign_landing_page.drupal.liquid [hunk 13]
```diff
@@ -317,12 +317,51 @@
...
+              {{ fieldClpConnectWithUs.entity.name }} Linkedin
+            </a>
+          </li>
+          <li class="vads-l-col--6 medium-screen:vads-l-col--4 vads-u-margin-bottom--2">
+            <a href="https://instagram.com/{{ socialLinksObject.instagram }}" onclick="recordEvent({ 'event': 'nav-hub-rail', 'nav-path': 'Connect with us' });" rel="noreferrer noopener" target="_blank">
```
Comment (timwright12) [inline src/site/layouts/campaign_landing_page.drupal.liquid:359]: same new window comment

File: src/site/layouts/campaign_landing_page.drupal.liquid [hunk 14]
```diff
@@ -317,12 +317,51 @@
...
+              <i class="fab fa-instagram vads-u-padding-right--1"></i>
```
Comment (timwright12) [inline src/site/layouts/campaign_landing_page.drupal.liquid:360]: same icon comment

File: src/site/layouts/campaign_landing_page.drupal.liquid (hunk 3 above)
Comment (kelsonic) [reply src/site/layouts/campaign_landing_page.drupal.liquid]: Thank you! 🙂 Updated!

File: src/site/layouts/campaign_landing_page.drupal.liquid (hunk 4 above)
Comment (kelsonic) [reply src/site/layouts/campaign_landing_page.drupal.liquid]: Thank you! 🙂 Updated!

**Review by timwright12 — APPROVED** [review]

**Pull request merged**